docs/data/material/getting-started/templates/dashboard/components/Navbar.tsx

```diff
@@ -1,122 +1,144 @@
 import { PaletteMode } from '@mui/material';
+import useMediaQuery from '@mui/material/useMediaQuery';
 import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
 import Stack from '@mui/material/Stack';
 import MuiToolbar from '@mui/material/Toolbar';
-import Tab from '@mui/material/Tab';
-import Tabs from '@mui/material/Tabs';
+import Typography from '@mui/material/Typography';
 import MenuRoundedIcon from '@mui/icons-material/MenuRounded';
 import NotificationsRoundedIcon from '@mui/icons-material/NotificationsRounded';
+import DashboardRoundedIcon from '@mui/icons-material/DashboardRounded';
 import ToggleColorMode from './ToggleColorMode';
-import SideNav from './SideNav';
+import SideMenuMobile from './SideMenuMobile';
 import MenuButton from './MenuButton';
-import NavbarBreadcrumbs from './NavbarBreadcrumbs';
 import OptionsMenu from './OptionsMenu';
+import Search from './Search';
 
 interface NavBarProps {
   mode: PaletteMode;
   toggleColorMode: () => void;
 }
 
 const Toolbar = styled(MuiToolbar)({
-  maxWidth: 1538,
   width: '100%',
-  padding: '16px 16px 0 16px',
+  padding: '12px',
   display: 'flex',
   flexDirection: 'column',
   alignItems: 'start',
   justifyContent: 'center',
   gap: '12px',
   flexShrink: 0,
-  backdropFilter: 'blur(24px)',
   '& .MuiTabs-flexContainer': {
     gap: '8px',
     p: '8px',
     pb: 0,
   },
 });
 
-function a11yProps(index: number) {
-  return {
-    id: `simple-tab-${index}`,
-    'aria-controls': `simple-tabpanel-${index}`,
-  };
-}
-
 export default function Navbar({ mode, toggleColorMode }: NavBarProps) {
   const [open, setOpen] = React.useState(false);
+  const theme = useTheme();
+  const isDesktop = useMediaQuery(theme.breakpoints.up('md'));
 
   const toggleDrawer = (newOpen: boolean) => () => {
     setOpen(newOpen);
   };
 
-  const [value, setValue] = React.useState(0);
-
-  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
-    setValue(newValue);
-  };
-
   return (
     <AppBar
       position="fixed"
-      sx={(theme) => ({
+      sx={{
+        display: { xs: 'auto', md: 'none' },
         boxShadow: 0,
-        bgcolor: 'transparent',
+        bgcolor: 'background.paper',
         backgroundImage: 'none',
-        alignItems: 'center',
         borderBottom: '1px solid',
-        borderColor: theme.palette.divider,
-      })}
+        borderColor: 'divider',
+      }}
     >
       <Toolbar variant="regular">
-        <Stack
-          direction="row"
-          sx={{
-            gap: 1,
-            alignItems: 'center',
-            justifyContent: { xs: 'flex-end', md: 'space-between' },
-            flexGrow: 1,
-            width: '100%',
-            display: { xs: 'none', md: 'flex' },
-          }}
-        >
-          <NavbarBreadcrumbs />
-          <Stack direction="row" sx={{ gap: 1 }}>
-            <MenuButton showBadge>
-              <NotificationsRoundedIcon />
+        {isDesktop ? (
```

Comment:
Previously, we used the `sx` prop with `display: { xs: 'none', md: 'flex' }` / `display: { sm: 'flex', md: 'none' }` for this toggle. What's the reason for the change?